keras_cv/layers/detectron2_layers.py

```diff
@@ -0,0 +1,431 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.backend import keras
+from keras_cv.backend import ops
+from keras_cv.models.segmentation.segment_anything.sam_layers import MLPBlock
+
+
+def get_rel_pos(query_size, key_size, rel_pos):
```

Comment:
Let's move the helper functions to the bottom of the file

Reply:
Addressed in [ac7f30e](https://github.com/keras-team/keras-cv/pull/1987/commits/ac7f30ee2cb0ee7569244cf67dd18a50ef805752)